pkg/visualizer/file.go

```diff
@@ -49,7 +50,7 @@ type (
 )
 
 func (f *File) Path() string {
-	return "topology.yaml"
+	return fmt.Sprintf("%stopology.yaml", f.PathPrefix)
```

Comment:
nit: should we be `filepath.Join()` here instead of a string format?

Reply:
the path prefix is really just a filename prefix so i will rename